src/frontend/magnify/src/components/rooms/RoomRow/RoomRow.tsx

```diff
@@ -123,6 +147,17 @@ export default function RoomRow({ room, baseJitsiUrl }: RoomRowProps) {
       ),
       onClick: openDeleteModal,
     });
+
+    result.push({
+      icon: <Copy />,
```

Comment:
Use the `<Clone />` icon instead. And pay attention to the size, it's not at all uniform with the other icons